### PR TITLE
Add AppImage

### PR DESCRIPTION
**Pull request by probonopd**
unicorn

### CODE REVIEW

**Comment by probonopd** [comment]
Please re-run the test.

I have fixed the "List item description must end with proper punctuation" issue at
https://github.com/sindresorhus/awesome/runs/3337854478?check_suite_focus=true#step:3:13
in 
https://github.com/AppImage/awesome-appimage/commit/3f8f1b68d4984c19ea8148c9c4656d4f2d80e50c

**Comment by msub2** [comment]
To add on to the same link issue I would suggest these resolutions:
- AppImage.GitHub.io - Keep the one on line 64, get rid of the one on line 189.
- appimage.org - Get rid of the one on line 269, there's no need to link it a second time when you've already done it at the very beginning of the list.
- linuxdeployqt - I would keep the first mention and look for another Travis CI example on a different repo or elsewhere.

**Comment by probonopd** [comment]
Please re-run the test.

I have fixed the (partially) duplicated URLs.

File: readme.md [hunk 1]
```diff
@@ -175,6 +175,7 @@
 	- [Containers](https://github.com/Friz-zy/awesome-linux-containers#readme)
 	- [eBPF](https://github.com/zoidbergwill/awesome-ebpf#readme) - Virtual machine that allows you to write more efficient and powerful tracing and monitoring for Linux systems.
 	- [Arch-based Projects](https://github.com/PandaFoss/Awesome-Arch#readme) - Linux distributions and projects based on Arch Linux.
+	- [AppImage](https://github.com/AppImage/awesome-appimage#readme) - Cross-distribution format and tools to package applications for Linux as single files.
```
Comment (sindresorhus) [inline readme.md:178]: I think the description could be shorter and clearer.

Maybe this?

> Package apps in a single file that works on various mainstream Linux distributions.

Comment (probonopd) [reply readme.md]: Thanks, done.

**Comment by sindresorhus** [comment]
Linting is failing

**Comment by saintmalik** [comment]
> Linting is failing

Looks like "Deployment tools for Flash applications","Deployment tools for Rust applications" ToC is missing

**Comment by probonopd** [comment]
Thanks, should be fixed now.
My lint now succeeds:
https://github.com/AppImage/awesome-appimage/runs/3643981695

**Comment by sindresorhus** [comment]
Please also enable the issue tracker on the repo.

**Comment by probonopd** [comment]
Done.

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1439511028824379393

**Pull request merged**